modules/virt-about-node-placement-virtualization-components.adoc

```diff
@@ -11,18 +11,22 @@ You might want to customize where {VirtProductName} deploys its components to en
 * Operators only deploy on infrastructure nodes.
 * Certain nodes are unaffected by {VirtProductName}. For example, you have workloads unrelated to virtualization running on your cluster, and you want those workloads to be isolated from {VirtProductName}.
 
-You can specify node placement rules for the {VirtProductName} Operators that Operator Lifecycle Manager (OLM) deploys by editing the OLM `Subscription` object.
+[id="how-to-apply-node-placement-rules-virt-components"]
+== How to apply node placement rules to virtualization components
 
-To specify node placement rules for components that the {VirtProductName} Operators deploy, edit the HyperConverged Cluster custom resource (CR).
+You can specify node placement rules for a component by editing the corresponding object directly or by using the web console.
 
-[id="about-node-placement-rules_{context}"]
-== About node placement rules
+* For the {VirtProductName} Operators that Operator Lifecycle Manager (OLM) deploys, edit the OLM `Subscription` object directly. Currently, you cannot configure node placement rules for the `Subscription` object by using the web console.
+* For components that the {VirtProductName} Operators deploy, edit the `HyperConverged` object directly or configure it by using the web console during {VirtProductName} installation.
+* For the hostpath provisioner, edit the `HostPathProvisioner` object directly or configure it by using the web console.
```

Comment:
Hi Pan,

Dalia discovered a [pitfall about using this feature that might be worth documenting in a line or two of text](https://bugzilla.redhat.com/show_bug.cgi?id=1919905), if you are OK with handling it in this PR.
If virtualization pods are scheduled on a node, but HPP isn't, then using virtualization pods with HPP will fail to run (it sounds obvious written this way :)).

Suggesting this text:

Note: hostpath provisioner needs to be scheduled on the same nodes as the virtualization components for correct functionality.

Reply:
Thanks for the review, @maya-r! I added a warning as follows: "You must schedule the hostpath provisioner and the virtualization components on the same nodes. Otherwise, virtualization pods that use the hostpath provisioner fail to run."

Does that sound good?

Reply:
Sounds good to me!